### ISSUE TITLE
field 'lastPosition'

### ISSUE THREAD

**Issue by HakanOnurM** [ESX-Org/es_extended#187]
New player joining and : 

```
Error running call reference function for resource es_extended: citizen:/scripting/lua/scheduler.lua:403: server/classes/player.lua:202: attempt to index a nil value (field 'lastPosition')
stack traceback:
        server/classes/player.lua:202: in field 'getLastPosition'
        server/main.lua:223: in upvalue 'cb'
        server/classes/player.lua:232: in upvalue 'ref'
        citizen:/scripting/lua/scheduler.lua:389: in function <citizen:/scripting/lua/scheduler.lua:388>
        [C]: in function 'xpcall'
        citizen:/scripting/lua/scheduler.lua:388: in function <citizen:/scripting/lua/scheduler.lua:387>
stack traceback:
        [C]: in function 'error'
        citizen:/scripting/lua/scheduler.lua:403: in function <citizen:/scripting/lua/scheduler.lua:372>
[ERROR] [MySQL] Check the error above, an error happens when executing the callback from the query : "SELECT * FROM `user_accounts` WHERE `identifier` = @identifier {@identifier=steam:11000011c50b199}"
```

**Comment by x1mart** [comment]
Same here since the last update

**Comment by murphyhf** [comment]
Same...

**Comment by ElPumpo** [comment]
fixed in 22a4d67971e3e0ed79d446cf37929cc8e28ff79b, sorry!
